keyring.go

```diff
@@ -231,8 +243,12 @@ func (k KeyRing) VerifyJSONs(ctx context.Context, requests []VerifyJSONRequest)
 
 		fetched, err := fetcher.FetchKeys(ctx, keyRequests)
 		if err != nil {
-			fetcherLogger.WithError(err).WithField("fetcher", fetcher.FetcherName()).
-				Warn("Failed to request keys from fetcher")
+			fetcherLogger.WithError(err).Warn("Failed to request keys from fetcher")
```

Comment:
We've lost the fetcher name here.

Reply:
`fetcherLogger` already includes that field, it was duplicated in this case.